addons/pos_restaurant/static/tests/tours/refund_tour.js

```diff
@@ -43,5 +43,9 @@ registry.category("web_tour.tours").add("RefundStayCurrentTableTour", {
             ProductScreen.isShown(),
             inLeftSide(ProductScreen.orderLineHas("Coca-Cola")),
             ProductScreen.totalAmountIs("-4.40"),
+            ProductScreen.clickPayButton(),
+            PaymentScreen.clickPaymentMethod("Cash"),
+            PaymentScreen.clickValidate(),
+            ReceiptScreen.clickNextOrder(),
```

Comment:
```suggestion
           ReceiptScreen.isShown(),
```
check ReceiptScreen is enough.